css/main.less

```diff
@@ -63,7 +63,6 @@ h1, h2, h3, h4, h5, h6 {
   color: @primary-brand;
   font-weight: 700;
   margin-bottom: 0.5em;
-  margin-top: 2em
```

Comment:
Why is this line removed? Will this affect the styles of other headers (i.e. elements with header tags) on the website?